matRad_calcParticleDose.m

```diff
@@ -194,12 +190,12 @@
                  && strcmp(pln.radiationMode,'carbon')
                     mTissueClass_j= mTissueClass(ix,:);
             end
-            
+              
             for k = 1:stf(i).numOfBixelsPerRay(j) % loop over all bixels per ray
-
+                
                 counter = counter + 1;
-                % Display progress
-                matRad_progress(counter,dij.totalNumOfBixels);
+                counterBeam = counterBeam + 1;
```

Comment:
I am somewhat confused. Why do we need a counterBeam here? isnt this a counter of bixelsPerBeam? Better rename...